cmd/create.go

```diff
@@ -224,6 +224,14 @@ Note: remove interval field from job specification for manually triggered jobs`,
 this effects runtime dependencies and template macros`,
 			},
 		},
+		{
+			Name: "metadata.resource",
+			Prompt: &survey.Confirm{
+				Message: "Do you want to configure resource?",
+				Default: false,
+				Help:    "Set configuration for CPU or Memory for each job",
+			},
+		},
```

Comment:
I don't think we should ask this as a question. This will be mostly used by advanced users and adding this in normal flow will only add confusion.

Reply:
So, are you suggesting to not show this question? Like, the user can configure it by manually updating the `job.yaml` file?

Reply:
Yes, they can manually add that in yaml.

Reply:
it's updated